docs/src/main/mdoc/certificates.md

```diff
@@ -0,0 +1,34 @@
+---
+id: security
+title: Security & Certificates
+---
+
+## Security: certificates, trust stores, and passwords 
+
+The `KafkaCredentialStore` can be used to create the necessary.
+
+The parameters passed in are string representations of the client private key, client certificate
+and service certificate. the `properties` field in `KafkaCredentialStore` can then be applied to
+any of the `*Settings` classes by using the `withProperties(kafkaCredentialStore.properties)`.
+
+```scala mdoc
+import cats.effect._
+import cats.syntax.all._
+import fs2.kafka.security._
+
+def loadKafkaSetup[F[_]: Async: ContextShift](
```

Comment:
I think this only needs `Sync` rather than `Async`?

Reply:
Good point will check it thanks